site/app/(home)/page.tsx

```diff
@@ -19,9 +20,9 @@ export default async function Home({ searchParams }: HomeProps) {
     <>
       <Hero title="Create your Player">
         Look on our player theme treasures; compatible with most web video and audio players.{' '}
-        <a href="/about" className="underline">
+        <Link href="/about" className="underline">
```

Comment:
Was this change necessary for post hog for some reason, or just a side fix?

Reply:
it wasn't required. a bit related in that Posthog adds the tracking to Next.js to accommodate pjax style navigation and that made me double check all the nav links were proper